project/Dependencies.scala

```diff
@@ -18,6 +18,8 @@ object Dependencies {
 
   val reflections = "net.oneandone.reflections8" % "reflections8" % "0.11.7"
 
+  val logger = "org.slf4j" % "slf4j-simple" % "1.7.30"
+
```

Comment:
Pls bump Borer to 1.7.1 for 2.13 only (if not easily doable then again, drop 2.12 support as for now)

Reply:
Opened https://github.com/sirthias/borer/issues/389, let's see what happens